### PR TITLE
feat: ability to toggle property visibility in config panel

### PR DESCRIPTION
**Pull request by Chandru-HM**
## Overview
added ability to toggle property visibility in config panel and it includes

1. property toggle now toggles the properties on the chart
2. mouse over shows 'hide' or 'unhide' tooltip wrt property visibility status and also shows opposite visibility icons 

## Legal
This project is available under the [Apache 2.0 License](http://www.apache.org/licenses/LICENSE-2.0.html).

### CODE REVIEW

File: packages/dashboard/src/customization/propertiesSections/propertiesAndAlarmsSettings/styledPropertyComponent.tsx [hunk 1]
```diff
@@ -88,30 +98,68 @@ export type StyledPropertyComponentProps = {
...
       <div style={{ fontWeight: 'normal' }}>{label}</div>
-      {/* <Button onClick={() => {}} variant='icon' iconSvg={StatusEyeVisible} /> TODO */}
+      <div className='property-display-toggle' onMouseEnter={handleMouseEnter} onMouseLeave={handleMouseLeave}>
+        <Button
+          onClick={onToggleAssetQuery}
+          variant='icon'
+          iconSvg={(!onMouseOver ? isPropertyVisible : !isPropertyVisible) ? StatusEyeVisible : StatusEyeHidden}
+        />
+        <span className='tooltiptext' style={tooltipStyle}>
+          {onMouseOver ? (isPropertyVisible ? 'hide' : 'unhide') : !isPropertyVisible ? 'hide' : 'unhide'}
```
Comment (corteggiano) [inline packages/dashboard/src/customization/propertiesSections/propertiesAndAlarmsSettings/styledPropertyComponent.tsx:159]: if the tooltip wont be visible once mouse leaves then why not just do 
`onMouseOver && (isPropertyVisible ? 'hide' : 'unhide')` ?

Comment (Chandru-HM) [reply packages/dashboard/src/customization/propertiesSections/propertiesAndAlarmsSettings/styledPropertyComponent.tsx]: done

**Review by corteggiano — APPROVED** [review]

File: packages/dashboard/src/customization/propertiesSections/propertiesAndAlarmsSettings/propertyComponent.css [hunk 2]
```diff
@@ -12,3 +12,48 @@
   display: flex;
   align-items: center;
 }
+
+.property-display-toggle {
+  position: relative;
+}
+
+.property-display-toggle .tooltiptext {
```
Comment (jmbuss) [inline packages/dashboard/src/customization/propertiesSections/propertiesAndAlarmsSettings/propertyComponent.css:20]: Is it possible to use this component for the tooltip? https://cloudscape.design/components/popover/?tabId=playground

File: packages/dashboard/src/customization/propertiesSections/propertiesAndAlarmsSettings/styledPropertyComponent.tsx [hunk 3]
```diff
@@ -88,30 +98,68 @@ export type StyledPropertyComponentProps = {
...
       {colorable && display === 'property' && (
         <ColorPicker color={property.color || ''} updateColor={(newColor) => updateStyle({ color: newColor })} />
       )}
       <div style={{ fontWeight: 'normal' }}>{label}</div>
-      {/* <Button onClick={() => {}} variant='icon' iconSvg={StatusEyeVisible} /> TODO */}
+      <div className='property-display-toggle' onMouseEnter={handleMouseEnter} onMouseLeave={handleMouseLeave}>
+        <Button
+          onClick={onToggleAssetQuery}
+          variant='icon'
+          iconSvg={(!onMouseOver ? isPropertyVisible : !isPropertyVisible) ? StatusEyeVisible : StatusEyeHidden}
```
Comment (jmbuss) [inline packages/dashboard/src/customization/propertiesSections/propertiesAndAlarmsSettings/styledPropertyComponent.tsx:156]: Is the idea with this ternary that the icon changes to reflect the next status? Can we make some variables to represent these states so we don't have nested ternaries?

File: packages/dashboard/src/customization/propertiesSections/propertiesAndAlarmsSettings/propertyComponent.css (hunk 2 above)
Comment (jmbuss) [reply packages/dashboard/src/customization/propertiesSections/propertiesAndAlarmsSettings/propertyComponent.css:20]: If this isn't possible, can we make some component for tooltips? I know we also have them in this component - `packages/dashboard/src/components/palette/icons/index.tsx`

Comment (Chandru-HM) [reply packages/dashboard/src/customization/propertiesSections/propertiesAndAlarmsSettings/propertyComponent.css:20]: no, popover component is not working as the expected. needs to open on mouse over

File: packages/dashboard/src/customization/propertiesSections/propertiesAndAlarmsSettings/styledPropertyComponent.tsx (hunk 3 above)
Comment (Chandru-HM) [reply packages/dashboard/src/customization/propertiesSections/propertiesAndAlarmsSettings/styledPropertyComponent.tsx:156]: 2 scenarios, icon change to reflect the next status and click should change as well. can you please elaborate your suggestion.

Comment (Chandru-HM) [reply packages/dashboard/src/customization/propertiesSections/propertiesAndAlarmsSettings/styledPropertyComponent.tsx]: done

**Pull request merged**